x/gov/handler_test.go

```diff
@@ -1177,8 +1417,33 @@ func withProposal(t *testing.T, db store.CacheableKVStore, ctx weave.Context, mo
 
 	return pBucket
 }
+func withElectorateUpdateProposal(t *testing.T, db store.KVStore, ctx weave.Context, mods ...ctxAwareMutator) *ProposalBucket {
+	// setup electorate
```

Comment:
You can call [`t.Helper()`](https://golang.org/pkg/testing/#T.Helper) at the beginning of the helper function to produce better error message in case of a failure.